aspnetcore/tutorials/grpc/grpc-start.md

```diff
@@ -9,8 +9,366 @@ uid: tutorials/grpc/grpc-start
 ---
 # Tutorial: Create a gRPC client and server in ASP.NET Core
 
-::: moniker range=">= aspnetcore-5.0"
+::: moniker range=">= aspnetcore-6.0"
+This tutorial shows how to create a .NET Core [gRPC](xref:grpc/index) client and an ASP.NET Core gRPC Server. At the end, you'll have a gRPC client that communicates with the gRPC Greeter service.
 
+In this tutorial, you:
```

Comment:
There isn't a link to download sample code like there is for .NET 5. Is that intentional?

Reply:
The .NET 6 version is there, but moved to the bottom. Rick requested I move it to the bottom here:

![image](https://user-images.githubusercontent.com/10985336/135171432-a3901612-f5f8-4461-9b7a-63cc2bfb9f59.png)

Reply:
Ok! Should .NET 5 download link be moved for consistency?

(I don't have a strong opinion either way)

Reply:
Sure. I will move the download link to the bottom for .NET 5, the same as the .NET 6 version. Thanks.

Reply:
Rick just mentioned leaving it at the top for .NET 5, so leaving it there.